libs/wire-api/src/Wire/API/User/Handle.hs

```diff
@@ -49,14 +53,30 @@ modelUserHandleInfo = Doc.defineModel "UserHandleInfo" $ do
   Doc.property "user" Doc.string' $
     Doc.description "ID of the user owning the handle"
 
+instance ToSchema UserHandleInfo where
+  declareNamedSchema _ = do
+    qualifiedIdSchema <- declareSchemaRef (Proxy @(Qualified UserId))
+    unqualifiedIdSchema <- deprecatedUnqualifiedSchemaRef (Proxy @UserId) "qualified_user"
+    pure $
+      NamedSchema
+        (Just "UserHandleInfo")
+        $ mempty
+          & type_ ?~ SwaggerObject
+          & properties
+            .~ [ ("user", unqualifiedIdSchema),
+                 ("qualified_user", qualifiedIdSchema)
+               ]
+
 instance ToJSON UserHandleInfo where
   toJSON (UserHandleInfo u) =
     object
-      ["user" .= u]
+      [ "user" .= _qLocalPart u, -- For backwards comptaibility
+        "qualified_user" .= u
```

Comment:
so here we use qualified_user not qualified_id? Isn't that inconsistent then?

Reply:
I am just prepending `qualified_` to whatever we had earlier. Doing this sounds more consistent than changing everything to `qualified_id`. WDYT?

Reply:
hm, I don't know. The old fields (sometimes called 'user' and sometimes called 'id') seemed inconsistent; this could be a time to change that and not keep that inconsistency. But maybe it doesn't matter if clients use the swagger definitions to create their API.